retrodeck.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# Steam Deck SD path: /run/media/mmcblk0p1
+
+is_mounted() {
+    mount | awk -v DIR="$1" '{if ($3 == DIR) { exit 0}} ENDFILE{exit -1}'
+}
+
+# if we got the .lock file it means that it's not a first run
+if [ ! -f ~/retrodeck/.lock ]
+then
+	kdialog --title "RetroDECK" --yes-label "Yes" --no-label "Quit" --yesno "Welcome to the first configuration of RetroDECK.\n\nBefore starting, are you in Desktop Mode?\nIf not the program will quit as the first setup MUST be done in Desktop Mode."
+	if [ $? == 1 ] #quit
+    then
+		exit 0
+	fi
+    kdialog --title "RetroDECK" --yes-label "Internal" --no-label "SD Card" --yesno "Where do you want your roms folder to be located?"
+    if [ $? == 0 ] #yes - Internal
+    then
+        roms_folder=~/retrodeck/roms
+    else #no - SD Card
+        if is_mounted "/run/media/mmcblk0p1"
+        then
+            roms_folder=/run/media/mmcblk0p1/retrodeck/roms
```

Comment:
this is not portable at all. Not even acros the same distro.

Reply:
No, it's only for Steam Deck as it should be.